compiler/modules/bir/bir.bal

```diff
@@ -188,6 +188,8 @@ public type InsnBase record {
     # Whether an instruction is a terminator or a PPI is determined
     # just by its name.
     InsnName name;
+    # Position based on the AST node that generated the instruction
+    Position pos;
```

Comment:
This is good, but you should include it in all the insns, even the ones that provide their own defninition of `name`

Reply:
fixed